src/resolvers/learningpathResolvers.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+  fetchLearningpath,
+  fetchLearningpathStep,
+  fetchResource,
+  fetchArticle,
+} from '../api';
+
+interface LearningpathArgument {
```

Comment:
Kunne benytta QueryToLearningpathArgs fra schema.d.ts i stedet for å definere interfacet på nytt her. Samme med QueryToLearningpathStepArgs  for neste interface.